src/reactRe.rei

```diff
@@ -10,9 +10,9 @@ type event =
     which : int,
     stopPropagation : (unit => unit) [@bs.meth],
     preventDefault : (unit => unit) [@bs.meth],
-    currentTarget : ReasonJs.Document.element,
-    relatedTarget : ReasonJs.Document.element,
-    target : ReasonJs.Document.element
+    currentTarget : ReasonJs.Element.t,
+    relatedTarget : ReasonJs.Element.t,
+    target : ReasonJs.Element.t
```

Comment:
I don't know why it would. It's not functionally a child of Document. If trying to model an inheritance tree, it would be more appropriate to have Element be a child of Node. But the DOM isn't really an inheritance tree either, it's more a of a DAG of interfaces or mixins.

For an API consumer it might initially make more sense that Element is nested under Document, but that will quickly get you in trouble. E.g. where would DocumentFragment fit in?